src/compose/network.ts

```diff
@@ -160,6 +160,15 @@ class NetworkImpl implements Network {
 			configOnly: network.config_only || false,
 		};
 
+		// Add label if there's non-default ipam config
+		// e.g. explicitly defined subnet or gateway.
+		// When updating between a release where the ipam config
+		// changes, this label informs the Supervisor that
+		// there's an ipam diff that requires recreating the network.
+		if (net.config.ipam.config.length > 0) {
+			net.config.labels['io.balena.network.ipam'] = 'true';
```

Comment:
I would like us to use the`io.balena.private` namespace, I would prefer something like `io.balena.private.ipam.config` perhaps? `io.balena.private.network.ipam` works too, although the `network` part might not totally be necessary since we are already configuring networks

Reply:
Changed to `io.balena.private.ipam.config`!